louis/db/api/__init__.py

```diff
@@ -1,5 +1,33 @@
-from louis.models import openai
+#from louis.models import openai
+import louis.models.openai as openai
+import os
+import json
+import sys
+import louis.db as db
+import dotenv
+# This is used to load the .env file
+dotenv.load_dotenv()
 
+FINESSE_WEIGHTS = os.environ.get("FINESSE_WEIGHTS") \
+    or db.raise_db_error("FINESSE_WEIGHTS is not set")
+
+# should be something like that : '{
+#     "recency": 1,
+#     "traffic": 1,
+#     "current": 0.5,
+#     "typicality": 0.2,
+#     "similarity": 1
+# }'
+
+if FINESSE_WEIGHTS:
+    try:
+        with open(FINESSE_WEIGHTS, 'r') as json_file:
+            json_data = json_file.read()
+        parsed_json = json.loads(json_data)
```

Comment:
This could be a constant. The naming should also be more descriptive of what it contains. Example: `FINESSE_WEIGHTS_JSON_DATA`

Reply:
done